introductions.md

```diff
@@ -1,22 +1,23 @@
 ### Transactions Quest
 
-*Sometimes science is more art than science. - Rick Sanchez*
+_Sometimes science is more art than science. - Rick Sanchez_
 
-Welcome and congratulations for choosing to learn blockchains and cryptocurrencies. 
+Welcome and congratulations for choosing to learn blockchains and cryptocurrencies.
 
 Today we get a first experience in the most core element, transactions. We will use the actual Bitcoin software to create transactions and send them. We will learn to interact with command lines, APIs and wallets to move cryptocurrencies around.
 
 Most open blockchains offer three categories of networks:
+
 - The main network `mainnet` on which value is transacted
 - One or several `testnet`, that work more or less similarly to the mainnet with no value
-- Local networks, to test locally with a node. It is called `regtest` on Bitcoin. 
+- Local networks, to test locally with a node. It is called `regtest` on Bitcoin.
 
-*🚨 Caution 🚨*
+_🚨 Caution 🚨_
 You should not use any crypto with value for any exercise of the module. If you already own crypto, we recommend that you use a new separate wallet to avoid any loss of funds. We will never use the main network of any blockchain `mainnet`.
 
 #### Mandatory
 
-1. [sendTransaction](sendTransaction/README.md) _Send a Bitcoin transaction_
+1. [sendTransaction](https://github.com/01-edu/public/blob/master/subjects/blockchain/send-transaction/README.md) _Send a Bitcoin transaction_
```

Comment:
Why was this the only link that changed?

Reply:
The document is not used anywhere (as far as I know) - I changed the link of this exercise only to have a PR related to 1 exercise only.

If you find it clearer, I can restore the `master` version and, in the future, update all the links in one PR.